target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -242,6 +242,12 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x2f20
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary info 0x8) +1)
 		;;
+	xiaomi,aiot-ac2350)
+		caldata_extract "art" 0x5000 0x2f20
+		ath10k_patch_mac $(mtd_get_mac_binary art 0x5006)
```

Comment:
This line can be dropped, the address will be read by the line before.

Reply:
Fixed in d332184